examples/user-notification.amp.html

```diff
@@ -229,6 +229,37 @@
      <button on="tap:amp-user-notification3.dismiss">I accept</button>
   </amp-user-notification>
 
+  <amp-user-notification
+          layout=nodisplay
+          id="amp-user-notification4"
+          always-show
+          data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
So, I missed bunch of naming discussion, but wanted to ask why don't we:

1. What don't we call it `persist="no"`. Negative flags are not that unusual, e.g. `scrolling=no`.
2. What don't we use `data-` prefix? If we decided to use `data-` in other attributes, wouldn't it follow to use it here as well?